src/content/inferno/js/InfernoRegion.ts

```diff
@@ -472,7 +482,8 @@ export class InfernoRegion extends Region {
       player.location = { x: 25, y: 15 };
 
       // spawn zuk
-      const shield = new ZukShield(this, { x: 23, y: 13 }, { aggro: player });
+      const shieldDirection = this.initializeAndGetShieldDirection();
+      const shield = new ZukShield(this, { x: 23, y: 13 }, { aggro: player }, shieldDirection);
```

Comment:
To avoid this repetition here in both `initialiseRegion` and `spawnNextWave` I guess the Zuk spawning code could be moved to a separate method but I avoided doing this to not bloat this PR with unrelated changes